operations/compare-helm-with-jsonnet/components/config/kustomization.yaml

```diff
@@ -45,6 +45,13 @@ patches:
       - op: remove
         path: /config/blocks_storage/bucket_store/metadata_cache
 
+  - target:
+      kind: MimirConfig
+      name: 'ruler|querier'
+    patch: |-
+      - op: remove
+        path: /config/blocks_storage/bucket_store/metadata_cache/memcached/max_idle_connections
+
```

Comment:
I've added this because in Helm same metadata cache settings are used between ruler, querier and store-gateways, while we only configure store-gateway in Jsonnet.